EXILED/EXILED.props

```diff
@@ -15,7 +15,7 @@
 
   <PropertyGroup>
     <!-- This is the global version and is used for all projects that don't have a version -->
-    <Version Condition="$(Version) == ''">8.14.0</Version>
```

Comment:
☠

Reply:
I don't know why this happened. It's almost my first time making a pull request. Sorry if something is wrong.

Reply:
Just remove editing this file (rollback or just change 1 back to 0)

Reply:
I'm a bit lost, can you explain it to me?